IPython/html/static/services/contents.js

```diff
@@ -7,7 +7,7 @@ define(function(require) {
     var $ = require('jquery');
     var utils = require('base/js/utils');
 
-    var Contents = function(options) {
+    var Contents = function(base_url, common_config) {
```

Comment:
No, I think it should stay as keyworks arguments (i.e. a dictionary). I just meant pass them explicitly.